firebase-app-distribution/src/test/java/com/google/firebase/appdistribution/FirebaseAppDistributionTest.java

```diff
@@ -236,6 +244,7 @@ public void updateApp_whenNotSignedIn_throwsError() throws Exception {
   @Test
   public void updateToLatestRelease_whenNewAabReleaseAvailable_showsUpdateDialog()
       throws Exception {
+    shadowOf(Looper.getMainLooper()).idle();
```

Comment:
Did we figure out if we need this? Or did adding the `addcancelationListener` fix the issue this was trying to solve

Reply:
We did not need this. removing now.